README.md

```diff
@@ -1,40 +1,376 @@
+- Install the CNPG plugin using kubectl krew (recommended):
+  ```bash
+  kubectl krew install cnpg
+  kubectl cnpg version
+  ```
+  > **Alternative installation methods:**
+  > - For Debian/Ubuntu: Download `.deb` from [releases page](https://github.com/cloudnative-pg/cloudnative-pg/releases)
+  > - For RHEL/Fedora: Download `.rpm` from [releases page](https://github.com/cloudnative-pg/cloudnative-pg/releases)
+  > - See [official installation docs](https://cloudnative-pg.io/documentation/current/kubectl-plugin) for all methods
+- Optional but recommended: `kubectx`, `stern`, `kubectl-view-secret` (see the [CNPG Playground README](https://github.com/cloudnative-pg/cnpg-playground#prerequisites) for a complete list).
+- **Disk Space:** Minimum **30GB** free disk space recommended:
+  - Kind cluster nodes: ~5GB
+  - Container images: ~5GB (first run with image pull)
+  - Prometheus/MongoDB storage: ~10GB
+  - Jepsen results + logs: ~5GB
+  - Buffer for growth: ~5GB
+- Sufficient local resources for a multi-node Kind cluster (≈8 CPUs / 12 GB RAM) and permission to run port-forwards.
+
+Once the tooling is present, everything else is managed via repository scripts and Helm charts.
+
+---
+
+## ⚡ Setup and Configuration
+
+> Follow these sections in order; each references the authoritative upstream documentation to keep this README concise.
+
+### 0. Clone the Chaos Testing Repository
+
+**First, clone this repository to access the chaos experiments and scripts:**
+
+```bash
+git clone https://github.com/cloudnative-pg/chaos-testing.git
+cd chaos-testing
+```
+
+All subsequent commands reference files in this repository (experiments, scripts, monitoring configs). Keep this terminal window open.
+
+### 1. Bootstrap the CNPG Playground
+
+The upstream documentation provides detailed instructions for prerequisites and networking. Follow the setup instructions here: <https://github.com/cloudnative-pg/cnpg-playground#usage>.
+
+**Open a new terminal** and run:
+
+```bash
+git clone https://github.com/cloudnative-pg/cnpg-playground.git
+cd cnpg-playground
+./scripts/setup.sh eu         # creates kind-k8s-eu cluster
+./scripts/info.sh             # displays contexts and access information
+export KUBECONFIG=$PWD/k8s/kube-config.yaml
+kubectl config use-context kind-k8s-eu
+```
+
+### 2. Install CloudNativePG and Create the PostgreSQL Cluster
+
+With the Kind cluster running, install/update the operator by following the official **CloudNativePG v1.27 Installation & Upgrades** guide (<https://cloudnative-pg.io/documentation/1.27/installation_upgrade/>). The snippets below mirror the documented steps:
+
+**In the cnpg-playground terminal:**
+
+```bash
+# Re-export the playground kubeconfig if you opened a new shell
+export KUBECONFIG=$PWD/k8s/kube-config.yaml
+kubectl config use-context kind-k8s-eu
+
+# Apply the 1.27.1 operator manifest exactly as documented
```

Comment:
Given that you have the plugin installed, I would rely on the plugin to install the latest version of the operator. See https://github.com/cloudnative-pg/cnpg-playground/blob/main/demo/setup.sh#L65